src/components/index.ts

```diff
@@ -1 +1,2 @@
 export { default } from './Navbar/Navbar';
+export * from './Loading/Loading';
```

Comment:
If you use a default export in the `Loading` component, your export here should look similar to the Navbar's export. 

For instance, you could do the following: 
```suggestion
export { default as Navbar} from './Navbar/Navbar';
export {default as Loading} from './Loading/Loading';
```